setup.py

```diff
@@ -3,8 +3,21 @@
 import shlex
 import glob
 import sys
+import os
 
-extensions = [Extension("geometry", sources=["src_c/geometry.c"])]
+
+compiler_options = {"unix": ["-mavx2"], "msvc": ["/arch:AVX2"]}
```

Comment:
This is not necessary. Since you're using intrinsics, the compiler isn't emitting the symbols, you are. With these options, the project will not run at all on older systems, I believe.